packages/expo/src/Notifications/Notifications.ts

```diff
@@ -370,5 +373,42 @@ export async function scheduleNotificationWithTimerAsync(
 }
 
 function isInRangeInclusive(variable: number, min: number, max: number): boolean {
-  return (variable >= min && variable <= max);
+  return variable >= min && variable <= max;
+}
+
+/*
+ * Legacy code
+ */
```

Comment:
What about describing when this `legacy code` could be dropped.
Like `used prior SDK33` or so 😄